openid-4-verifiable-presentations-1_0.md

```diff
@@ -611,6 +629,20 @@ The Wallet then validates the request as specified in OAuth 2.0 [@RFC6749].
 
 If the Verifier responds with any HTTP error response, the Wallet MUST terminate the process.
 
+## Attachments in Authorization Requests {#attachments-in-authorization-requests}
+
+The attachment data mechanism enables to pass additional information to the Wallet to request credentials. The Verifier MAY include attachments in the Authorization Request, for example to pass a registration certificate to inform about the registered intended use or an authorization attestation to request a specific credential from the Wallet to comply with the embedded disclosure policy.
+
+### Proof of Possession
+
+In some cases, the Verifier may want to include attachments that are bound to a key or require verification of their origin.
```

Comment:
isn't that the normal case?
And what does verification of origin mean? Origin of the credential or origin of the request?

Reply:
This phrase was still there when it was not limited to the credential request. In this case, POP is relevant for all attestations, so I will update it.

Reply:
Updated to make it mandatory